src/main/webapp/index.html

```diff
@@ -17,25 +17,23 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
     <title>CodeU 2019 Starter Project</title>
     <link rel="stylesheet" href="/css/main.css">
     <script src="/js/navigation-loader.js"></script>
   </head>
   <body onload="addLoginOrLogoutLinkToNavigation();">
     <nav>
+    <div class="dropdown"style="float:right;">
```

Comment:
is the float:right supposed to be here or should it be under the dropdown stuff with the css?

Reply:
I think it can go both ways, but this way made it easier for me to understand. I was having issues with trying to offset the image, but switching to float on the HTML side worked without so much of a hitch.